### PR TITLE
Added and verified security checks.

### PR DESCRIPTION
**Pull request by prabathariyaratna**

### CODE REVIEW

File: modules/distribution/src/main/assembly/bin.xml [hunk 1]
```diff
@@ -281,6 +281,21 @@
...
+            <destName>web.xml</destName>
+        </file>
+        <file>
+            <source>src/main/conf/carbon/web.xml</source>
+            <outputDirectory>wso2esb-${pom.version}/repository/conf/tomcat/carbon/WEB-INF</outputDirectory>
+            <destName>web.xml</destName>
+        </file>
+        <file>
+            <source>src/main/conf/webapp-mode/web.xml</source>
+            <outputDirectory>wso2esb-${pom.version}/webapp-mode/WEB-INF</outputDirectory>
```
Comment (kasunbg) [inline modules/distribution/src/main/assembly/bin.xml]: AFAIK, webapp-mode is no longer used. This was used to deploy Carbon as a webapp in other application servers.

Comment (prabathariyaratna) [inline modules/distribution/src/main/assembly/bin.xml]: Those XML files were already there. We just override this files to adding some extra elements for avoiding security vulnerabilities.

Comment (kasunbg) [inline modules/distribution/src/main/assembly/bin.xml]: Ok. If we don't use it, you don't really need to update it. Anyway, we should get rid of this folder in the next release.

Comment (prabathariyaratna) [inline modules/distribution/src/main/assembly/bin.xml]: +1 Removing, We updated this files for safeguard :)

**Pull request merged**